### PR TITLE
boresight calibration

### PR DESCRIPTION
**Pull request by ToivoS**

### CODE REVIEW

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml [hunk 1]
```diff
@@ -157,9 +157,9 @@
...
-    launch_file_input_bimanual: "sr_teleop_vive_polhemus.launch sim:=true vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input_right: "sr_teleop_vive_polhemus.launch sim:=true side:=right vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml:160]: remove calibration_file_name var, rename calibration_file_path to boresight_calibration_file_path

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml [hunk 2]
```diff
@@ -157,9 +157,9 @@
...
+    launch_file_input_left: "sr_teleop_vive_polhemus.launch sim:=true side:=left vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml:161]: also here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml [hunk 3]
```diff
@@ -157,9 +157,9 @@
...
+    launch_file_input_bimanual: "sr_teleop_vive_polhemus.launch sim:=true vive:={{ real_vive | lower }} biotacs:={{ biotacs | lower }} robot_model:={{ ur_robot_type }} polhemus:={{ real_glove | lower }} palm_device:=tracker require_pedal:=false user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml:162]: also here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml [hunk 4]
```diff
@@ -204,8 +204,8 @@
 
 - name: Set Shadow Glove Launch file inputs if real_glove = true
   set_fact:
-    launch_file_input_right: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
-    launch_file_input_left: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input_right: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml:207]: same here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml [hunk 5]
```diff
@@ -204,8 +204,8 @@
...
+    launch_file_input_left: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml:208]: same here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml [hunk 6]
```diff
@@ -198,7 +198,7 @@
     desktop_icon_png: "{{ teleop_glove_right }}"
     launch_script: "shadow_GUI_right.sh"
     project_name_input: "sr_teleop_vive_polhemus"
-    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=true vive:=true robot_model:={{ ur_robot_type }} require_pedal:=true biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=true vive:=true robot_model:={{ ur_robot_type }} require_pedal:=true biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml:201]: remove calibration_file_name var, rename calibration_file_path to boresight_calibration_file_path

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml [hunk 7]
```diff
@@ -224,7 +224,7 @@
     desktop_icon_png: "{{ teleop_glove_left }}"
     launch_script: "shadow_GUI_left.sh"
     project_name_input: "sr_teleop_vive_polhemus"
-    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=true side:=left vive:=true robot_model:={{ ur_robot_type }} require_pedal:=true biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=true side:=left vive:=true robot_model:={{ ur_robot_type }} require_pedal:=true biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml:227]: also here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml [hunk 8]
```diff
@@ -265,7 +265,7 @@
     desktop_icon_name: "5 - Launch Right Shadow Glove Driver"
     desktop_icon_path: "Shadow Advanced Launchers/5 - Launch Right Shadow Glove Driver"
     project_name_input: "sr_teleop_vive_polhemus"
-    launch_file_input: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml:268]: also here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml [hunk 9]
```diff
@@ -278,7 +278,7 @@
     desktop_icon_name: "5 - Launch Left Shadow Glove Driver"
     desktop_icon_path: "Shadow Advanced Launchers/5 - Launch Left Shadow Glove Driver"
     project_name_input: "sr_teleop_vive_polhemus"
-    launch_file_input: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml:281]: also here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml [hunk 10]
```diff
@@ -291,7 +291,7 @@
     desktop_icon_name: "5 - Launch Left Shadow Glove Driver"
     desktop_icon_path: "Shadow Advanced Launchers/5 - Launch Bimanual Shadow Glove Driver"
     project_name_input: "sr_fingertip_hand_teleop"
-    launch_file_input: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input: "polhemus.launch user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml:294]: also here

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml [hunk 11]
```diff
@@ -430,7 +430,7 @@
     desktop_icon_name: "Right Shadow Glove Teleop Simulation"
     desktop_icon_path: "Right Shadow Glove Teleop Simulation"
     project_name_input: "sr_teleop_vive_polhemus"
-    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=false sim:=true vive:={{ real_vive | lower }} robot_model:={{ ur_robot_type }} require_pedal:=false biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=false sim:=true vive:={{ real_vive | lower }} robot_model:={{ ur_robot_type }} require_pedal:=false biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```
Comment (giusebar) [inline ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml:433]: and here

**Review by giusebar — CHANGES_REQUESTED** [review]

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml (hunk 1 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml (hunk 3 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml (hunk 4 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml (hunk 5 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml (hunk 6 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml (hunk 8 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml (hunk 7 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml (hunk 9 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml (hunk 10 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml (hunk 11 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml]: Fixed in the last commit

File: ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml (hunk 2 above)
Comment (ToivoS) [reply ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml]: Fixed in the last commit

**Review by joannaphillips45 — APPROVED** [review]

**Review by giusebar — APPROVED** [review]

**Pull request merged**